ui/src/cloud/apis/orgsettings.ts

```diff
@@ -0,0 +1,15 @@
+import {OrgSettings} from 'src/types'
+import {getAPIBasepath} from 'src/utils/basepath'
+
+export const getOrgSettings = async (orgID: string): Promise<OrgSettings> => {
+  try {
+    const response = await fetch(
+      `${getAPIBasepath()}/api/v2private/orgs/${orgID}/settings`
+    )
+    const data = await response.json()
```

Comment:
shouldn't we be looking for a status code here as well, just in case an error message comes back as valid json?

Reply:
Data is data. As long as what we are looking for is there, why not? 😄 

But you make a a good point. I will improve the handling.

Reply:
We'll go with status 200 as the only valid status. Any other status, including redirects would result in an "error" and we will shout at you to **_OMG UPGRADE NOW_** with shiny buttons.